media/css/base/mozid-modal.less

```diff
@@ -0,0 +1,152 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+html.noscroll body {
+    overflow: hidden;
+}
+
+#modal {
+    background: #000;
+    background: rgba(0, 0, 0, 0.85);
+    width: 100%;
+    height: 101%;
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 9999999;
```

Comment:
I know this is just copy/pasta from the old one, but do you think we should add another digit on here, just to be extra safe? :trollface: